src/RestSharp/Enum.cs

```diff
@@ -19,9 +20,44 @@ namespace RestSharp;
 /// </summary>
 public enum ParameterType {
     /// <summary>
-    /// Cookie parameter
+    /// A <see cref="Parameter"/> that will added to the QueryString for GET, DELETE, OPTIONS and HEAD requests; and form for POST and PUT requests.
     /// </summary>
-    GetOrPost, UrlSegment, HttpHeader, RequestBody, QueryString
+    /// <remarks>
+    /// See <see cref="GetOrPostParameter"/>.
+    /// </remarks>
+    GetOrPost,
+
+    /// <summary>
+    /// A <see cref="Parameter"/> that will be added to part of the url by replacing a {placeholder} within the absolute path.
```

Comment:
Utilizing the `c` tag for `{placeholder}` will allow documentation generators to use code tags:
```csharp
/// A <see cref="Parameter"/> that will be added to part of the url by replacing a <c>{placeholder}</c> within the absolute path.
```

Reply:
What exactly would this do?

Reply:
It's the `<code>` tag that would change the typeface when the code inside the tag is rendered in the popup.